### ISSUE TITLE
Option title isn't showing 

### ISSUE THREAD

**Issue by Ain477** [snapappointments/bootstrap-select#2570]
On version 1.13 the option title was showing after upgrading to 1.14 it's not showing any more, could you please add it back?

![Screenshot from 2021-02-04 17-33-56](https://user-images.githubusercontent.com/26634947/106887382-fc7fbc00-66dc-11eb-9f10-ba0b7db0b449.png)

Here it should be showing only **[100]** instead of the whole text as per the previous version.

**Comment by qant** [comment]
Screen shot is not prove any thing. Please share codepen with bug example. Thank you.

**Comment by kursadcayir** [comment]
Hi,
`<select class="selectpicker" data-title="select Title"> <option title="t1" value="1">OPTION1</option> <option title="t2" value="2">OPTION2</option> <option title="t3" value="3">OPTION3</option> <option title="t4" value="4">OPTION4</option> </select>`
I created that HTML and put it on my screen, until now I am using selectpicker with bootstrap5 with no problem but this not work like the explanation in the docs about title option.
Thanks in advance
ps : bootstrap-select-1.14.0-beta2 is the version and bootstrap5 beta3

**Comment by soid** [comment]
Yes, `title` attribute for `<option>`  is broken in bootstrap-select-1.14.0-beta2 and bootstrap 5.0.0

**Comment by giarico** [comment]
I can confirm the issue.
On line 1956 option.title is always undefined.
option.option.title contains the correct value.
